src/main/java/net/snowflake/ingest/streaming/internal/RowBufferStats.java

```diff
@@ -398,8 +395,8 @@ void addStrValue(String inputValue) {
     String value =
         inputValue.length() > MAX_LOB_LEN ? inputValue.substring(0, MAX_LOB_LEN) : inputValue;
 
-    byte[] valueBytes = value != null ? value.getBytes(StandardCharsets.UTF_8) : null;
-    byte[] collatedValueBytes = value != null ? getCollatedBytes(value) : null;
```

Comment:
why remove the null check?

Reply:
This is recommended by Intellj as well because we already do `inputValue.length` above, but I don't think this is correct since we could add a NULL string, will update

Reply:
Actually, `convertRowToArrow` already have a separate branch for NULL so we're good